axlearn/common/trainer.py

```diff
@@ -253,7 +253,7 @@ def __init__(
         self._per_param_train_dtype = maybe_instantiate(
             canonicalize_per_param_dtype(cfg.train_dtype)
         )
-
+        self._maybe_record_event(measurement.Event.START_ACCELERATOR_INIT)
```

Comment:
Is there a guideline on this event? I am not very sure where `START_ACCELERATOR_INIT` should be added

Reply:
The design doc I shared with you previously should provide more insight on this. (Specifically, the *Badput Breakdown, Accelerator Initialization Time* section)

Reply:
Thanks. It is probably just much easier to understand if we treat the whole "__init__" function as "_ACCELERATOR_INIT" event.

Imagine someone wants to add some more code in `__init__`. They will need to think if they should add it before `self._maybe_record_event(measurement.Event.START_ACCELERATOR_INIT)` or after.